tests/paths.rs

```diff
@@ -0,0 +1,38 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at http://mozilla.org/MPL/2.0/.
+
+extern crate hglib;
+
+use crate::hglib::{hg, paths};
+use std::fs::OpenOptions;
+use std::io::Write;
+
+mod common;
+
+#[test]
+fn test_basic() {
+    let mut c = common::TestClient::new("paths_basic", &[]);
+
+    let path = c.get_path(".hg/hgrc");
+
+    let mut file = OpenOptions::new().append(true).open(path).unwrap();
```

Comment:
you can probably just use `c.append(".hg/hgrc",&["[paths]", "foo = bar"])`

Reply:
done!